lte/gateway/c/session_manager/SetMessageManagerHandler.cpp

```diff
@@ -86,17 +86,18 @@ void SetMessageManagerHandler::SetAmfSessionContext(
      */
     // Requested message from AMF to release the session
     if (cfg.common_context.sm_session_state() == RELEASED_4) {
-      if (cfg.common_context.sm_session_version() != 0) {
+      if (cfg.common_context.sm_session_version() == 0) {
         MLOG(MERROR) << "Wrong version received from AMF for IMSI " << imsi
                      << " but continuing release request";
+      } else {

```

Comment:
is `" but continuing release request"` valid anymore. It looks like before there were not an `else`

Reply:
removed that  2nd line as we are not proceeding further.